tests/Unit/Jobs/UpdateAllProductsTest.php

```diff
@@ -64,68 +61,216 @@ public function setUp(): void {
 			$this->merchant_center
 		);
 
+		$this->action_scheduler
+			->method( 'has_scheduled_action' )
+			->willReturn( false );
+
+		$this->merchant_center
+			->method( 'is_connected' )
+			->willReturn( true );
+
+		/* adding a filter to make batch smaller for testing */
+		add_filter(
+			'woocommerce_gla_batched_job_size',
+			function ( $batch_count, $job_name ) {
+				if ( self::JOB_NAME === $job_name ) {
+					return 2;
+				}
+				return $batch_count;
+			}, 10, 2 );
```

Comment:
If we split the parameters to multiple lines then the PHPCS rules we use require us to put each parameter on a separate line.
```suggestion
			},
			10,
			2
		);
```